apps/pl/scss/_scss2json.scss

```diff
@@ -3,6 +3,8 @@
 // Import Everything from Base.
 // This includes Bootstrap
 @import '../../../source/_patterns/00-protons/variables';
```

Comment:
Noting here that we're going to update scss `includePaths` to make this much nicer.

Reply:
Is that something to add to this PR or in the future?

Reply:
No need in this PR. I'm going to do some pretty large Sass architecture cleanup soon.